docs/qcfractal/source/install.rst

```diff
@@ -1,30 +1,35 @@
 Install QCFractal
 =================
 
-You can install qcfractal with ``conda``, with ``pip``, or by installing from source.
+You can install qcfractal with ``conda`` (recommended) or with ``pip`` (with some caveats).
+
+The below commands install QCFractal and its required dependencies, but *not* any of the quantum
+chemistry codes nor the software to run :term:`Queue Managers <Manager>`. This is done to avoid requiring *all* software
+which *can* interface with Fractal, and instead requires the user to obtain the software they individually *require*.
+
+.. _conda-install:
 
 Conda
 -----
 
-You can install or update qcfractal using `conda <https://www.anaconda.com/download/>`_::
+You can install qcfractal using `conda <https://www.anaconda.com/download/>`_:
 
-    conda install qcfractal -c conda-forge
+.. code-block:: console
 
-The above command installs qcfractal and its required dependencies, but *not* any of the quantum
-chemistry codes nor the software to run :term:`Queue Managers <Manager>`. This is done to avoid requiring *all* software
-which *can* interface with Fractal, and instead requires the user to obtain the software they individually *require*.
+    >>> conda install qcfractal -c conda-forge
 
-The qcfractal package is maintained on the
+This installs QCFractal and its dependencies. The qcfractal package is maintained on the
```

Comment:
We sometimes say QCFractal and qcfractal. Should we standardize to "\`\`qcfractal\`\`" here?